### DISCUSSION TITLE
IPv6 public node

### DISCUSSION THREAD

**Discussion by deavmi** [qaul/qaul.net#652] [Ideas]
Would be good to get an IPv6 variant of your community node running. I only have
IPv6 (clearnet) connectivity where I am and would like to test it

**Comment by MathJud** [comment]
The qaul public node supports all currently supported protocols (TCP & QUIC) via IPv4 & IPv6.
The IPv6 address is `2a02:c207:3004:3887::1` the port number remains the same: `9229`

## Configure IPv6 public node in the qaul App

To change the currently configured public node to IPv6, go to the `settings` menu and edit the existing 

## Configure IPv6 for CLI 

If you are running qaul from the CLI, edit the configuration file `config.yaml`.
Change the entry `/ip4/144.91.74.192/udp/9229/quic-v1` to one of the values below:

```
/ip6/2a02:c207:3004:3887::1/udp/9229/quic-v1
/ip6/2a02:c207:3004:3887::1/tcp/9229
```

**Comment by deavmi** [comment]
Hi, thanks!

May I recommend that this ship with the Android app as one of the default nodes (alongside the IPv4 one)?

On 12 November 2024 14:38:30 SAST, Mathias Jud ***@***.***> wrote:
>The qaul public node supports all currently supported protocols (TCP & QUIC) via IPv4 & IPv6.
>The IPv6 address is `2a02:c207:3004:3887::1` the port number remains the same: `9229`
>
>## Configure IPv6 public node in the qaul App
>
>To change the currently configured public node to IPv6, go to the `settings` menu and edit the existing 
>
>## Configure IPv6 for CLI 
>
>If you are running qaul from the CLI, edit the configuration file `config.yaml`.
>Change the entry `/ip4/144.91.74.192/udp/9229/quic-v1` to one of the values below:
>
>```
>/ip6/2a02:c207:3004:3887::1/udp/9229/quic-v1
>/ip6/2a02:c207:3004:3887::1/tcp/9229
>```
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/qaul/qaul.net/discussions/652#discussioncomment-11225633
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>

----
Tristan B. Velloza Kildaire

Web: https://deavmi.assigned.network
BNET IRC: deavmi
Email: ***@***.***

**Reply by MathJud** [reply]
Just added :) [e41f862](https://github.com/qaul/qaul.net/commit/e41f86277c9874f621d2428303760432e7673e0b)
Will be in the next release

**Comment by deavmi** [comment]
Thanks!

On 13 November 2024 18:17:38 SAST, Mathias Jud ***@***.***> wrote:
>Just added :) [e41f862](https://github.com/qaul/qaul.net/commit/e41f86277c9874f621d2428303760432e7673e0b)
>Will be in the next release
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/qaul/qaul.net/discussions/652#discussioncomment-11242602
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>

----
Tristan B. Velloza Kildaire

Web: https://deavmi.assigned.network
BNET IRC: deavmi
Email: ***@***.***
